app/src/main/res/values/dimens.xml

```diff
@@ -587,5 +587,7 @@
   <dimen name="section_title_text_view_layout_margin_start">28dp</dimen>
   <dimen name="section_title_text_view_layout_margin_top">28dp</dimen>
   <dimen name="section_title_text_view_layout_margin_end">32dp</dimen>
+  <!-- Story Fragment dimens. -->
```

Comment:
Just name it as `<!-- StoryFragment -->`

Reply:
okk, I will change it.